integration/src/test/scala/org/knora/webapi/E2ESpec.scala

```diff
@@ -9,27 +9,38 @@ import akka.http.scaladsl.client.RequestBuilding
 import akka.http.scaladsl.model._
 import akka.testkit.TestKitBase
 import com.typesafe.scalalogging._
-import dsp.errors.FileWriteException
-import org.knora.webapi.config.AppConfig
-import org.knora.webapi.core.{AppRouter, AppServer, TestStartupUtils}
-import org.knora.webapi.messages.store.sipimessages.SipiUploadResponse
-import org.knora.webapi.messages.store.triplestoremessages.{RdfDataObject, TriplestoreJsonProtocol}
-import org.knora.webapi.messages.util.rdf._
-import org.knora.webapi.routing.{KnoraRouteData, UnsafeZioRun}
-import org.knora.webapi.testservices.{FileToUpload, TestClientService}
-import org.knora.webapi.util.{FileUtil, LogAspect}
 import org.scalatest.BeforeAndAfterAll
 import org.scalatest.concurrent.ScalaFutures
 import org.scalatest.matchers.should.Matchers
 import org.scalatest.wordspec.AnyWordSpec
 import spray.json._
 import zio._
 
-import java.nio.file.{Files, Path, Paths}
+import java.nio.file.Files
+import java.nio.file.Path
+import java.nio.file.Paths
 import java.util.concurrent.TimeUnit
-import scala.concurrent.{Await, ExecutionContext, Future}
+import scala.concurrent.Await
+import scala.concurrent.ExecutionContext
+import scala.concurrent.Future
 import scala.concurrent.duration.FiniteDuration
 
+import dsp.errors.FileWriteException
+import org.knora.webapi.config.AppConfig
+import org.knora.webapi.core.AppRouter
+import org.knora.webapi.core.AppServer
+import org.knora.webapi.core.TestStartupUtils
+import org.knora.webapi.messages.store.sipimessages.SipiUploadResponse
+import org.knora.webapi.messages.store.triplestoremessages.RdfDataObject
+import org.knora.webapi.messages.store.triplestoremessages.TriplestoreJsonProtocol
+import org.knora.webapi.messages.util.rdf._
+import org.knora.webapi.routing.KnoraRouteData
+import org.knora.webapi.routing.UnsafeZioRun
+import org.knora.webapi.testservices.FileToUpload
+import org.knora.webapi.testservices.TestClientService
+import org.knora.webapi.util.FileUtil
+import org.knora.webapi.util.LogAspect
```

Comment:
I think this is the regular scalafix behaviour as defined in `.scalafix.conf`: 1. block external dependencies, 2. block standard library dependencies, 3. block project internal dependencies. Each block is alphabetically ordered.

We can of course change these configurations, but that's outside the scope of this task.  
The only reason this shows up here, is because my PR fixes our `fmt` command :)

Reply:
No worries, this makes sens. Thanks!